ext/exif/exif.c

```diff
@@ -14,6 +18,10 @@ VALUE rb_exif_const_missing(VALUE klass, VALUE name) {
   return rb_call_super(1, &name);
 }
 
+VALUE rb_libexif_version() {
+  return rb_str_new2(LIBEXIF_VERSION_S);
```

Comment:
The version number is a constant, which means we don't need to create a new instance every time. How about replacing `Exif.lib_exiv_version` with `Exif::EXIF_VERSION`? I think the latter makes more sense because we already have `Exif::VERSION`.

Reply:
Good point. This is done, using `Exif::LIBEXIF_VERSION`